heft-plugins/heft-sass-plugin/src/SassProcessor.ts

```diff
@@ -235,8 +241,37 @@ export class SassProcessor extends StringValuesTypingsGenerator {
       result = await compileStringAsync(fileContents, {
         importers: [
           {
-            findFileUrl: (url: string): URL | null => {
-              return this._patchSassUrl(url, nodeModulesUrl);
+            // eslint-disable-next-line @typescript-eslint/naming-convention
+            findFileUrl: async (url: string, context: CanonicalizeContext): Promise<URL | null> => {
+              const packagePath: string = url.slice(1);
+              if (url[0] === '~') {
+                const { containingUrl } = context;
+                if (containingUrl) {
+                  let packageName: string;
+                  let modulePath: string;
+                  const firstSlashIndex: number = packagePath.indexOf('/');
+                  if (packagePath[0] === '@') {
+                    const secondSlashIndex: number = packagePath.indexOf('/', firstSlashIndex + 1);
+                    packageName = packagePath.slice(0, secondSlashIndex);
+                    modulePath = packagePath.slice(secondSlashIndex + 1);
+                  } else {
+                    packageName = packagePath.slice(0, firstSlashIndex);
+                    modulePath = packagePath.slice(firstSlashIndex + 1);
+                  }
```

Comment:
Could probably just have a var for `sliceIndex` and just use that, instead of calling `slice` under both conditions in the if-statement